test/EndToEnd/EndToEndTest.cs

```diff
@@ -244,7 +244,8 @@ private bool IsCentOS()
 
                 if(File.Exists(OSIDFILE))
```

Comment:
Could this not use `PlatformServices.Default.Runtime.OperatingSystem`? That property value is derived from the same file.